growth/too/tasks/scheduler.py

```diff
@@ -225,3 +230,78 @@ def get_decam_dict(data_row, queue_name, cnt, nrows,
     decam_dict["wait"] = "False"
 
     return decam_dict
+
+
+def get_growthindia_table(json_data, sunrise_hor=-12, horizon=20,
+                          priority=10000, domesleep=100):
+    """Make .csv file in GIT toO format for a given .json file"""
+    t = Table(rows=json_data['targets'])
+    coords = SkyCoord(ra=t['ra'], dec=t['dec'], unit=(u.degree, u.degree))
+    hanle = EarthLocation(lat=32.77889*u.degree,
+                          lon=78.96472*u.degree,
+                          height=4500*u.m)
+    iao = Observer(location=hanle, name="GIT", timezone="Asia/Kolkata")
+
+    twilight_prime = iao.sun_rise_time(Time.now(), which="next",
+                                       horizon=sunrise_hor*u.deg) - 12*u.hour
+    targets_rise_time = iao.target_rise_time(twilight_prime,
+                                             coords,
+                                             which="nearest",
+                                             horizon=horizon*u.degree)
+    targets_set_time = iao.target_set_time(targets_rise_time, coords,
+                                           which="next",
+                                           horizon=horizon*u.degree)
+    rise_time_IST = (targets_rise_time + 5.5*u.hour).isot
+    set_time_IST = (targets_set_time + 5.5*u.hour).isot
+    tend = targets_set_time
+    mooncoords = get_moon(tend, hanle)
+    sep = mooncoords.separation(coords)
+    dic = {}
+    dic['x'] = ['']*len(t)
+    dic['u'] = ['']*len(t)
+    dic['g'] = ['']*len(t)
+    dic['r'] = ['']*len(t)
+    dic['i'] = ['']*len(t)
+    dic['z'] = ['']*len(t)
+    target = ['EMGW']*len(t)
+    bands = {1: 'g', 2: 'r', 3: 'i', 4: 'z', 5: 'u'}
+
+    for i in range(len(t)):
+        filt = bands[t[i]['filter_id']]
+        dic[filt][i] = '1X%i' % (t[i]['exposure_time'])
+
+    del t['request_id', 'filter_id', 'program_pi', 'program_id',
+          'subprogram_name', 'exposure_time']
+    t['field_id'].name = 'tile_id'
+    t['dec'].name = 'Dec'
+    domesleeparr = np.zeros(len(t)) + domesleep
+    priority = np.zeros(len(t)) + priority
+    minalt = AltitudeConstraint(min=horizon*u.degree)
+    always_up = is_always_observable(minalt, iao, coords, Time(twilight_prime,
+                                     twilight_prime+12*u.hour))
+    rise_time_IST[np.where(always_up)] = (twilight_prime +
+                                          5.5*u.hour).isot
+    set_time_IST[np.where(always_up)] = (twilight_prime +
+                                         24*u.hour +
+                                         5.5*u.hour).isot
```

Comment:
This part is giving some errors - rise_time_IST and set_time_IST are both currently ISOT strings.  Looks like rise_time_IST and set_time_IST need to be arrays for this part of the code to work.

Reply:
Hi Shreya,

rise_time_IST and set_time_IST has same length as the rows in json data fed to this function. You can see this in line no. 247-255. I have independently checked this function via giving a json file data as input. It's working fine. I have used an older file that I received from Michael during followup of S190426C. If you have new file please sen dit to me. So, that I can recheck it. 

It would be great if you can share the error screenshot. This will help in debugging.

Reply:
Ah, in the example I was testing there was only one row in the json data from the gwemopt-generated schedule, so instead of an array of strings, rise_time_IST returned a single string. Could you make a small modification to your code setting 
rise_time_IST = np.array([rise_time_IST])
and likewise for set_time_IST, if coords has only one row? That should take care of the issue. I'm having some issues sending the json file here, so I'll send via slack.

Reply:
Will do that today itself.

Reply:
Will do that today itself.

Reply:
Will do that today itself.

Reply:
Could you set rise_time_IST and set_time_IST to numpy arrays right after they are originally defined, instead (i.e. around line 256)?  Line 282, where you are indexing these arrays is where the code is throwing an error.